install.sh

```diff
@@ -85,55 +159,92 @@ if !([ -f /etc/ros/rosdep/sources.list.d/20-default.list ]); then
 fi
 rosdep update
 
-
-#=====================================#
-# Workspace and Sub Repository Set Up #
-#=====================================#
+instalog "Installing Sub8 ROS dependencies"
+sudo apt-get install -qq binutils-dev
+sudo apt-get install -qq ros-kinetic-ompl
+sudo apt-get install -qq ros-kinetic-spacenav-node
+
+# Get older version of Google protobuf for Gazebo
```

Comment:
Have you confirmed with others that this is necessary, or the best way to do it?